sparse/usr/bin/droid/droid-hal-shutdown.sh

```diff
@@ -35,15 +35,15 @@
 # Kill all processes that are in this same cgroup.
 # Deducing the name of the service's cgroup based on the shutdown script's
 # cgroup name.
-CGROUP=$(cat /proc/self/cgroup | sed -r '/1:name=systemd:/!d;s|||;s|/control||')
-[ ! -f /sys/fs/cgroup/systemd/$CGROUP/cgroup.procs ] && echo "No such cgroup: $CGROUP" && exit 1
+CGROUP=$(sed -r '/1:name=systemd:/!d;s|||;s|/control||' < /proc/self/cgroup)
+[ ! -f "/sys/fs/cgroup/systemd/$CGROUP/cgroup.procs" ] && echo "No such cgroup: $CGROUP" && exit 1
```

Comment:
There's no need to quote this won't contain spaces.
Use if instead control-operators for more than one statement.

Reply:
Changed the code to use `if` statement.